bedrock/tabzilla/templates/tabzilla/tabzilla.js

```diff
@@ -351,29 +343,13 @@ var Tabzilla = (function (Tabzilla) {
     };
     Infobar.prototype.trackEvent = function (action, label, value,
                                              nonInteraction, callback) {
-        if (typeof(_gaq) !== 'object') {
-            if (callback) {
-                callback();
-            }
-
+        if (typeof(window.gaTrack) !== 'function') {
```

Comment:
We can't depend on `window.gaTrack` here, as this file is used on other sites, but...

Didn't we decide to leave this file unaltered? Unless I'm missing something, this file shouldn't be changed.